core/node/api_server/src/execution_sandbox/execute.rs

```diff
@@ -69,8 +69,8 @@ impl SandboxAction {
 }
 
 /// Output of [`SandboxExecutor::execute_in_sandbox()`].
-#[derive(Debug, Clone)]
-pub(crate) struct SandboxExecutionOutput {
+#[derive(Debug, Clone, Default)]
```

Comment:
Deriving `Default` is semantically incorrect here; for at least `are_published_bytecodes_ok`, the default value (`false`) means unsuccessful execution. Please use an inherent constructor instead (e.g. named `mock_success`). Would also allow to not derive `Default` for `ExecutionResult`, which I feel is semantically ambiguous (an empty return value is the default successful value for transactions, but not necessarily for calls).